html/aboutMe.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="/Css/aboutMe.css">
+    <title>About Me</title>
+</head>
+
+<body>
+        <header>
+            <div class="navigation_bar">
+
+                <img src="/Img/Netflix.png" alt="" width="200" height="50">
+                <a href="/index.html">INICIO</a>
+                <a href="/html/history.html">HISTORY</a>
+                <a href="/html/contact.html">CONTACT</a>
+                <a href="/html/aboutMe.html">MY PROFILE</a>
+
+            </div>
+        </header>
+
+        <h1>About Me</h1>
+
+        <div class="aboutMe">
+            <div class="content">
+
+            
+            <img class="Image" src="/Img/image_Valen.png" alt="Image about me" width="300px" height="300px">
+            <p>I am 18 years old. I am a very shy person, many confuse that with being "believed" however it is a word
+                that does not really define me, I like to study. My biggest weakness in study topics is that I never ask
+                because of nerves and it is something that should change, however I try to understand so as not to have
+                inconsistencies with important topics. I really like pop music, I really like Video Games, lately I play
+                a lot of valorant with my friends in my spare time and lastly I love spaghetti.</p>
+
+            <h1>skills and abilities</h1>
+
+            <p>I stand out a lot for being a person who can learn easily if I put my mind to it, I am a skilled,
+                responsible and academically good person. As time goes by, I can carry out a task quite quickly if I
+                have the tools and skills well worked out, I have good techniques in whatever I do and it is within my
+                reach.</p>
+        </div>
+        </div>
+
+        <div class="socialMedia">
+            <div class="content">
+
+            <a target="_blank" href="https://github.com/"><img src="/Img/GitHubpngwing.com.png" alt="Git Hub profile"
+                    width="150px" height="120px"></a>
+
+            <a target="_blank" href="https://www.linkedin.com/in/valen-ram%C3%ADrez-a7b517204/"><img
+                    src="/Img/Linpngwing.com.png" alt="LinkedIn profile" width="150px" height="120px"></a>
+
+                </div>
+                
+                <p id="Gmail">To know more contact me <a target="_blank" href="mailto:ramirezvalentina783@gmail.com">Ramirezvalentina783@gmail.com</a></p>
```

Comment:
this tags are too hard to read, your childrens should be inside the parent in the next line.